packages/grid/x-data-grid-premium/package.json

```diff
@@ -44,8 +44,8 @@
   "dependencies": {
     "@babel/runtime": "^7.18.9",
     "@mui/utils": "^5.10.3",
-    "@mui/x-data-grid": "5.17.10",
-    "@mui/x-data-grid-pro": "5.17.10",
+    "@mui/x-data-grid": "5.17.11",
+    "@mui/x-data-grid-pro": "5.17.11",
     "@mui/x-license-pro": "5.17.0",
```

Comment:
There is a change for `license` module. Shouldn't it have also been included in the release? 🤔